test/Table_Tests/src/In_Memory/Split_Tokenize_Spec.enso

```diff
@@ -204,7 +205,7 @@ add_specs suite_builder =
             t = Table.new cols
             expected_rows = [[0, "a", "c", Nothing, Nothing], [1, "c", "d", "ef", Nothing], [2, "gh", "ij", "u", Nothing]]
             expected = Table.from_rows ["foo", "bar 1", "bar 2", "bar 3", "bar 3"] expected_rows
-            t2 = t.split_to_columns "bar" "b" column_count=4
+            t2 = t.split_to_columns "bar" "b" column_count=(Columns_To_Add.First 4)
```

Comment:
Will the old behaviour still work? I think that it should and we should leave the test in place to ensure that it does

Reply:
Old behaviour will still work as we have conversion from Integer to Columns_To_Add.
We should keep a test proving this.